src/stories/components/custom-table-header/custom-table-header.stories.tsx

```diff
@@ -1,21 +1,46 @@
-import React from 'react';
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
 import { SortEnum } from '../../../core/enums/sort.enum';
 import { CustomTableHeader } from './custom-table-header';
-import type { CustomTableHeaderProps } from './custom-table-header';
-import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import type { ComponentMeta } from '@storybook/react';
 
 export default {
-  title: 'Components/General/CustomTableHeader',
+  title: 'Components/CuTable/CustomTableHeader',
   components: CustomTableHeader,
 } as ComponentMeta<typeof CustomTableHeader>;
 
-const Template: ComponentStory<typeof CustomTableHeader> = (args: CustomTableHeaderProps) => (
-  <CustomTableHeader {...args} />
-);
+const variantsArgs = [
+  {
+    title: 'Core Units',
+    state: SortEnum.Asc,
+    align: 'flex-start',
+    style: { marginLeft: '62px' },
+  },
+];
 
-export const Default = Template.bind({});
-Default.args = {
-  title: 'Some header',
-  state: SortEnum.Neutral,
-  align: 'flex-start',
+export const [[Header, HeaderDarkMode]] = createThemeModeVariants(CustomTableHeader, variantsArgs);
+
+Header.parameters = {
+  figma: {
+    component:
+      'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=6646%3A61302&t=iDXzm6LhfULmvnWw-4',
+    options: {
+      style: {
+        top: 20,
+        left: 6,
+      },
```

Comment:
Please, note that in this case the component and the image don't overlap

Reply:
done